### PR TITLE
Add Piskel.app v0.14.0

### PR DESCRIPTION
**Pull request by vuolter**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/piskel.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask 'piskel' do
+  version '0.14.0'
+  sha256 'c3f3e4fa5ac9495346dc30640b1e371062224bb4bb6eb26c5c3569151bff01e6'
+
+  # drive.google.com was verified as official when first introduced to the cask
+  url 'https://drive.google.com/uc?export=download&id=1xLt5JbaEoA1fMJgsfbnuD9KMMbuEysbj'
+  name 'Piskel'
```
Comment (vitorgalvao) [inline Casks/piskel.rb:7]: Please add an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md): `https://github.com/piskelapp/piskel/releases.atom`.

File: Casks/piskel.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+cask 'piskel' do
+  version '0.14.0'
+  sha256 'c3f3e4fa5ac9495346dc30640b1e371062224bb4bb6eb26c5c3569151bff01e6'
+
+  # drive.google.com was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/piskel.rb:5]: This needs to have the full URL. The current one is insufficient.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Welcome.

For future submissions, please do not untick the box to allow edits from maintainers. This could have already been fixed and merged.

File: Casks/piskel.rb (hunk 2 above)
Comment (vuolter) [reply Casks/piskel.rb]: Updated as requested.

File: Casks/piskel.rb (hunk 1 above)
Comment (vuolter) [reply Casks/piskel.rb:8]: Updated as requested. Note that latest release is `v0.15.0`, but the author has not yet published the binaries.

**Comment by vuolter** [comment]
> For future submissions, please do not untick the box to allow edits from maintainers. This could have already been fixed and merged.

Yeah, understood. I've updated the code. :)

**Comment by vitorgalvao** [comment]
Thank you for the submission, and the fast turnaround.

**Pull request merged**